lib/puppet_x/ldapquery.rb

```diff
@@ -129,7 +129,7 @@ def parse_entries
           attr = attribute.to_s
           value_data = []
           Array(values).flatten.each do |v|
-            value_data << v.chomp
+            value_data << v.chomp.to_s
```

Comment:
If it's an integer, you should first convert it to a string before calling `chomp`.
```suggestion
            value_data << v.to_s.chomp
```